components/layout_2020/dom_traversal.rs

```diff
@@ -448,10 +460,31 @@ where
                             vec.push(PseudoElementContentItem::Replaced(replaced_content));
                         }
                     },
+                    ContentItem::OpenQuote | ContentItem::CloseQuote => {
+                        // TODO(xiaochengh): calculate quote depth
+                        let depth: usize = 0;
+                        let quote: Option<String> = match &pseudo_element_style.get_list().quotes {
```

Comment:
Can the type be omitted here. If so, please do that.

Reply:
Done